src/main/java/com/icthh/xm/uaa/web/rest/TenantPropertiesResource.java

```diff
@@ -77,4 +82,19 @@ public UaaValidationVM validate(@RequestBody String uaaYml) {
     public PublicSettings getUaaPublicSettings() {
         return tenantPropertiesService.getTenantProps().getPublicSettings();
     }
+
+    @GetMapping(value = "/uaa/properties/data-schema/{roleKey}")
+    @ApiOperation(value = "Get role data schema", response = UserSpec.class)
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Uaa public settings", response = UserSpec.class),
+            @ApiResponse(code = 500, message = "Internal server error")})
+    @Timed
+    @PreAuthorize("hasPermission({'roleKey': #roleKey}, 'UAA.TENANT.PROPERTIES.UPDATE')")
```

Comment:
`UAA.TENANT.PROPERTIES.UPDATE` why the name is UPDATE?

Reply:
fixed